Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,184 @@
+name: pradeo-mtd
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: metadata
+  - name: initiator
+  - name: permission
+  - name: detection
+  - name: correlation
+  - name: compliance
+stages:
+  metadata:
+    actions:
+      - set:
+          pradeo.metadata.id: '{{json_event.message.id}}'
+          pradeo.metadata.creationDate: '{{json_event.message.creationDate}}'
+          pradeo.metadata.source: '{{json_event.message.source}}'
+          pradeo.metadata.category: '{{json_event.message.category}}'
+          pradeo.metadata.type: '{{json_event.message.type}}'
+  initiator:
+    actions:
+      - filter: '{{json_event.message.source == "device" and json_event.message.device != null}}'
+        set:
+          pradeo.initiator.id: '{{json_event.message.device.id}}'
+          pradeo.initiator.serialNumber: '{{json_event.message.device.serialNumber}}'
+          pradeo.initiator.name: '{{json_event.message.device.name}}'
+          pradeo.initiator.email: '{{json_event.message.device.email}}'
+          pradeo.initiator.imei: '{{json_event.message.device.imei}}'
+          pradeo.initiator.byod: '{{json_event.message.device.byod}}'
+          pradeo.initiator.declaredOperatingSystem: '{{json_event.message.device.declaredOperatingSystem}}'
+          pradeo.initiator.declaredOperatingSystemVersion: '{{json_event.message.device.declaredOperatingSystemVersion}}'
+          pradeo.initiator.declaredOperatingSystemSecurityPatchDate: '{{json_event.message.device.declaredOperatingSystemSecurityPatchDate}}'
+          pradeo.initiator.declaredModel: '{{json_event.message.device.declaredModel}}'
+          pradeo.initiator.lastConnection: '{{json_event.message.device.enrollmentStatus.lastConnection}}'
+          pradeo.initiator.coupled: '{{json_event.message.device.enrollmentStatus.coupled}}'
+          pradeo.initiator.mdmId: '{{json_event.message.device.emmDeviceInfo.externalId}}'
+          pradeo.initiator.emm: '{{json_event.message.device.emmDeviceInfo.emm}}'
+      - filter: '{{json_event.message.source == "admin" and json_event.message.user != null}}'
+        set:
+          pradeo.initiator.id: '{{json_event.message.user.id}}'
+          pradeo.initiator.email: '{{json_event.message.user.email}}'
+          pradeo.initiator.firstName: '{{json_event.message.user.firstName}}'
+          pradeo.initiator.lastName: '{{json_event.message.user.lastName}}'
+  permission:
+    actions:
+      - filter: '{{json_event.message.type == "DevicePermissionStatusUpdated"}}'
+        set:
+          pradeo.device.id: '{{json_event.message.content.device.id}}'
+          pradeo.device.serialNumber: '{{json_event.message.content.device.serialNumber}}'
+          pradeo.device.name: '{{json_event.message.content.device.name}}'
+          pradeo.device.email: '{{json_event.message.content.device.email}}'
```

[`user.email`](https://www.elastic.co/guide/en/ecs/8.11/ecs-user.html#field-user-email) is more suitable.
```suggestion
          user.email: '{{json_event.message.content.device.email}}'
```